src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/auditing/AuditableEvent.kt

```diff
@@ -31,13 +31,13 @@ data class AuditableEvent(
       )
     }
 
-    fun createLogInEvent(authentication: Authentication) = createEvent(AuditEventType.LOG_IN, authentication)
+    fun logInEvent(authentication: Authentication) = createEvent(AuditEventType.LOG_IN, authentication)
```

Comment:
Don't think we really need create in the name.  When using the service we see something like this anyway:

`auditService.create(AuditableEvent.someEvent())`